Framework/API/src/FrameworkManager.cpp

```diff
@@ -57,11 +43,7 @@
 #endif
 
 namespace {
-#ifdef TBB_HAS_GLOBAL_CONTROL
-std::unique_ptr<tbb::global_control> m_globalTbbControl;
-#else
-thread_local std::unique_ptr<tbb::task_scheduler_init> m_globalTbbControl;
-#endif
```

Comment:
I think it could be wise to add a comment here or someone is likely to put this back to some sort of scoped pointer in the future. Maybe something like:

```cpp
// We use a raw pointer over a unique_ptr to avoid problems with static deallocation order of static
// variables. In certain circumstances, unit testing, the shutdown method on FrameworkManager is not
// called. In this situation unique_ptr causes the tbb::global_control object to be destroyed
// *after* its own internal static data has been destroyed and we see a random segfault.
// We have technically introduced a leak by not deleting this object in the situation where
// `FrameworkManager::shutdown` is not called but the
// memory cannot be used again by the program since it survives for the duration of the
// program. It will ultimately be reclaimed by the OS.
```